### PR TITLE
PR to add kubectl feature;important for installing providers

### PR DESCRIPTION
**Pull request by patrickmukumbu**
# Description

PR to add kubectl feature;important for installing providers
## Features/Fixes/Patches/Changes list:

Please check the [CHANGELOG.md](https://github.com/boldlink/terraform-helm-release/blob/feature/kubectl-feature/CHANGELOG.md#110---2022-09-09)

### PR Owners Checklist:
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you checked there aren't open [Pull Requests](../../../pulls) of upstream or parallel code that will cause conflicts?
* [x] Have you updated the `CHANGELOG.md`?
* [x] Have you updated the `README.md`(s)?
* [ ] OWNER: Does your submission pass?
    * [x] Pre-commit (attach logs/print screen to this PR)
    * [x] Checkov/terrascan (attach logs/print screen to this PR)
    * [ ] Terraform resources examples build/destroy successfully with Terraform 0.14.11 minimum?
* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?
* [ ] First PR? Have you deleted `## How to use this template...` from the root `.README.md`?

### PR Reviewers Checklists?
* [ ] Are all your comments/changes resolved?
* [ ] Are you happy with the OWNER checklists and additional information provided?
* [ ] Does your review tests pass?
    * [ ] Terraform resources examples build/destroy successfully with Terraform 0.14.11 minimum?

### CODE REVIEW

File: examples/complete/main.tf [hunk 1]
```diff
@@ -15,4 +14,10 @@ module "complete_helm" {
       value = true
     },
   ]
+
+  manifests = [
+    {
+      yaml_body = file("ngnx.yaml")
```
Comment (hugoalmeida264) [inline examples/complete/main.tf:20]: Should be `nginx`

File: examples/complete/versions.tf [hunk 2]
```diff
@@ -6,5 +6,9 @@ terraform {
       source  = "hashicorp/helm"
       version = ">=2.1"
     }
+    kubectl = {
```
Comment (hugoalmeida264) [inline examples/complete/versions.tf:9]: Why are we using this custom kubectl version? Doesnt the official one work?

File: main.tf [hunk 3]
```diff
@@ -59,3 +58,26 @@ resource "helm_release" "main" {
     }
   }
 }
+
+## This resource is equivalent to `kubectl apply -f filename`. Use this to apply multiple single-document yaml files
+resource "kubectl_manifest" "main" {
```
Comment (hugoalmeida264) [inline main.tf:63]: Why are we adding the kubectl manifest here?

**Review by hugoalmeida264 — CHANGES_REQUESTED** [review]

**Comment by patrickmukumbu** [comment]
Closing this PR because the feature is included as an ad-hoc resource in labs-platform

**Pull request closed without merging**